zipkin-ui/js/page/default.js

```diff
@@ -27,12 +27,15 @@ const DefaultPageComponent = component(function DefaultPage() {
       const limit = query.limit || this.attr.config('queryLimit');
       const minDuration = query.minDuration;
       const endTs = query.endTs || new Date().getTime();
+      const defaultStartTs = 24 * 60 * 60 * 1000; // set default startTs 1 day ago;
```

Comment:
this should actually help a lot of sites not ask for so many traces. cc @jcarres-mdsol